keyboards/zsa/moonlander/moonlander.c

```diff
@@ -80,7 +80,10 @@ void moonlander_led_task(void) {
 #endif
 #if !defined(MOONLANDER_USER_LEDS)
     else {
-        layer_state_set_kb(layer_state);
+        static layer_state_t state = 0;
+        if (layer_state != state) {
+            layer_state_set_kb(layer_state);
```

Comment:
Need to set the local layer state somewhere:
```suggestion
            state = layer_state_set_kb(layer_state);
```

Reply:
Not sure how I missed this or how it worked for me in my testing, maybe because of timing luck.